website/docs/r/nginx_deployment.html.markdown

```diff
@@ -99,8 +107,12 @@ The following arguments are supported:
 
 * `network_interface` - (Optional) One or more `network_interface` blocks as defined below. Changing this forces a new Nginx Deployment to be created.
 
+* `scaling` - One or more `scaling` blocks as defined below.
```

Comment:
why do we need multiple scaling blocks?

Reply:
That is the schema pattern for the provider. It allows you to specify multiple blocks and the first one is what is actually used. 

I think we should refactor this in the future, but did not want to figure out an entirely new pattern for just this work.